src/components/container/SurveyTaskList/SurveyTaskList.tsx

```diff
@@ -9,13 +9,20 @@ import { ColorDefinition, resolveColor } from '../../../helpers/colors'
 import { ButtonVariant } from '../../presentational/Button/Button'
 import { useInitializeView } from '../../../helpers/Initialization';
 
+export interface SortBehavior {
```

Comment:
Since this is specific to this component, it is probably worth renaming to `SurveyTaskListSortBehavior` to set it apart from any generic `SortBehavior` type or interface we may define at some point.

The type could probably be exported separately, too, so it can be used in the Storybook or elsewhere.
```
export type SurveyTaskListSortBehaviorType = 'alphabetical' | 'dueDate' | 'insertedDate' | 'userDefined' | 'shuffle';

export interface SurveyTaskListSortBehavior {
	type: SurveyTaskListSortBehaviorType;
	direction?: SortOrder;
	userDefinedOrder?: string[];
}
```